src/frontend/src/lib/components/ui/ButtonAsIcon.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+	export let button: HTMLButtonElement | undefined = undefined;
+	export let testId: string | undefined = undefined;
+	export let ariaLabel: string;
+</script>
+
+<button
+	class="w-16 flex flex-col text-primary text-xs font-normal text-center"
+	bind:this={button}
+	on:click
+	aria-label={ariaLabel}
```

Comment:
In terms of accessibility I suggest to replicate the pattern I use for example in Juno: https://github.com/junobuild/juno/blob/main/src/frontend/src/lib/components/ui/ButtonIcon.svelte

Reply:
you mean, passing the label slot anyway but hide it? if that is the case, should the aria-label stay?

Reply:
Yes `<span class="visually-hidden"><slot /></span>`

Not sure I found it in this blog post but, few details about it: https://www.sarasoueidan.com/blog/accessible-icon-buttons/#technique-%231%3A-accessible-visually-hidden-text

Reply:
I think I understood! I copied the same schema as Juno (just in case we will use that `visually-hidden` class somehwere else), have a look please

Reply:
Coolio. I don't think there is something similar in Tailwind out-of-the-box so makes sense to me.